common/components/CheckboxItem/CheckboxItem.module.scss

```diff
@@ -0,0 +1,24 @@
+@use "common/abstracts" as *;
+
+.container {
+  display: flex;
+  border: 2px solid $grey;
+  border-radius: 8px;
+  padding: 0.75em 1.5em;
+}
+
+.container:hover {
+  border-color: $teal;
+  background-color: $light-teal;
```

Comment:
Good to have a hover state applied, but kinda liked to have this border and background for the checked/selected state. Perhaps we could keep either the border or background for the hover state?

Reply:
will now use border for hover state and border and background for checked.